source/documentation/other-topics/mandatory-aws-tags-update.html.md.erb

```diff
@@ -0,0 +1,66 @@
+---
+title: Updating Mandatory Tags for AWS provider in Cloud Platform namespace resources
+last_reviewed_on: 2025-03-11
+review_in: 6 months
+layout: google-analytics
+---
+
+# <%= current_page.data.title %>
+
+### Overview
+
+This guide explains how to update the mandatory tags for AWS provider in Cloud Platform namespace resources. This is essential for compliance and resource management of AWS resources.
+
+### Prerequisites
+
+Before you begin, ensure you have the following:
+
+- Access to the Cloud Platform namespace where the resources are located.
+- Latest version of the Cloud Platform CLI installed.
+
+### Steps to Update Mandatory AWS Tags
+
+1. **Running the cloud-platform CLI command to update tags:**
+    
+    Use the following command to update the mandatory AWS tags for your namespace resources:
+
+    ```bash
+    cloud-platform environment namespace-tags --namespaces namespace --repo-path /path/to/cloud-platform-environments
+    ```
+
+    Replace `namespace1` with the actual namespaces you want to update, and `/path/to/cloud-platform-environments` with the path to your Cloud Platform environments repository.
```

Comment:
In the command example the fag parameter is `namespace` not `namespace1`, and agai with ref to namespace[s] comment above, this sentence also suggests multiple namespaces can be passed?